modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/app/App.es.js

```diff
@@ -15,8 +15,8 @@
 import {openToast} from 'frontend-js-web';
 import core from 'metal';
 import dom from 'metal-dom';
-import {App} from 'senna';
 
+import {App} from '../../senna/senna';
```

Comment:
We should probably rename `senna/senna.js` to `senna/index.js`, or just `senna.js`, so that we could do this import as `from '../../senna'`);

Reply:
Yup!